recipes/diligent-core/all/conanfile.py

```diff
@@ -131,6 +132,8 @@ def _configure_cmake(self):
         self._cmake.definitions["DILIGENT_NO_GLSLANG"] = not self.options.with_glslang
         self._cmake.definitions["SPIRV_CROSS_NAMESPACE_OVERRIDE"] = self.options["spirv-cross"].namespace
         self._cmake.definitions["BUILD_SHARED_LIBS"] = False
+        self._cmake.definitions["DILIGENT_CLANG_COMPILE_OPTIONS"] = ""
+        self._cmake.definitions["DILIGENT_MSVC_COMPILE_OPTIONS"] = ""
```

Comment:
With https://github.com/DiligentGraphics/DiligentCore/commit/43ae908b68c3fe140e0caebd02b7bf04311c211e this may not be necessary